assets/src/block-editor/helpers/index.js

```diff
@@ -374,13 +374,19 @@ setUpInspectorControls.propTypes = {
 /**
  * Get AMP Layout select control.
  *
+ * @deprecated As of v2.1. Blocks with the `ampLayout` attribute will still be able to use the control.
+ *
  * @param {Object} props Props.
  *
  * @return {ReactElement} Element.
  */
 const AmpLayoutControl = ( props ) => {
 	const { name, attributes: { ampLayout }, setAttributes } = props;
 
+	if ( ! ampLayout ) {
```

Comment:
See my comment here https://github.com/ampproject/amp-wp/pull/5574/files#r518529646. We probably want this check to be for undefined instead of false.

Reply:
Good catch. Addressed in https://github.com/ampproject/amp-wp/pull/5573/commits/5993490a508b4c51517274bdc7c4b2f7cce600f2.